ImmunityReborn/Assets/_TeamAmnesia/Scripts/SettingsManager.cs

```diff
@@ -7,55 +7,42 @@ public class SettingsManager : MonoBehaviour
     public Toggle softcoreToggle;
     public Dropdown difficultyDropdown;
     private int previousDifficultyIndex = -1;
+    public GUISoundManager soundManager;
 
     void Start()
     {
-        difficultyDropdown.onValueChanged.AddListener( delegate { DifficultyChanged( difficultyDropdown.value ); } );
-        softcoreToggle.onValueChanged.AddListener(delegate { ToggleSoftcore(softcoreToggle.isOn); });
-        godModeToggle.onValueChanged.AddListener( delegate { ToggleGodMode( godModeToggle.isOn ); } );
+
```

Comment:
This was causing a conflict on starting the game because the listener set in the hierarchies were passing 0 values explicitly that conflicted with the actual values of the UI elements

Result: On game start, despite playerprefs being correct and the UI element showing the correct value, in actuality the difficulty was set to normal and the toggles were all off.